### PR TITLE
Provide descriptions for each method

### PR DESCRIPTION
**Pull request by Manishearth**
fixes https://github.com/immersive-web/webxr/issues/676

cc @ddorwin

r? @NellWaliczek

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -634,8 +637,9 @@ When requested, the {{XRSession}} MUST <dfn>apply the pending render state</dfn>
 </div>
 
 <div class="algorithm" data-algorithm="request-reference-space">
+The <dfn method for="XRSession">requestReferenceSpace(|type|)</dfn> method constructs a new {{XRReferenceSpace}} for a given |type|, if possible.
```
Comment (toji) [inline index.bs]: > a new {{XRReferenceSpace}} **for** a given |type|

Grammatical nit: Should this be "**of** a given type"?

File: index.bs [hunk 2]
```diff
@@ -920,7 +928,9 @@ The <dfn attribute for="XRFrame">session</dfn> attribute returns the {{XRSession
 
 <div class="algorithm" data-algorithm="get-viewer-pose">
 
-When the <dfn method for="XRFrame">getViewerPose(|referenceSpace|)</dfn> method is invoked, the user agent MUST run the following steps:
+The <dfn method for="XRFrame">getViewerPose(|referenceSpace|)</dfn> method provides the pose of the viewer relative to |referenceSpace| as an {{XRViewerPose}}, at the time represented by the {{XRFrame}}.
```
Comment (toji) [inline index.bs]: Please link-ify `[=viewer=]`

**Comment by Manishearth** [comment]
Done

**Comment by toji** [comment]
Thank you!

**Pull request merged**